synapse_data_warehouse/synapse/procs/R2.13.0__add_list_downloaders.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR REPLACE PROCEDURE list_downloaders(start_record_date string, entity_list string)
```

Comment:
Could we use the `DATE` type here?  string is also fine, but you can do something like

`record_date > DATE('2024-02-16')`